src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -72,6 +72,35 @@ public void ProcessStart_UseShellExecute_OnLinux_ThrowsIfNoProgramInstalled()
             }
         }
 
+        [Fact]
+        [OuterLoop("Opens program")]
+        public void ProcessStart_DirectoryNameInCurDirectorySameAsFileNameInExecDirectory_Success()
+        {
+            string fileToOpen = "dotnet";
+            string curDir = Environment.CurrentDirectory;
+            string dotnetFolder = Path.Combine(Path.GetTempPath(),"dotnet");
+            bool shouldDelete = !Directory.Exists(dotnetFolder);
+            try
+            {
+                Directory.SetCurrentDirectory(Path.GetTempPath());
+                Directory.CreateDirectory(dotnetFolder);
+
+                using (var px = Process.Start(fileToOpen))
```

Comment:
Not sure you can assume that dotnet is on the path. Maybe you need to append the test run folder to the path?

Reply:
I dont assume it's in the path. It is definitely in the testhost execution directory. Because it's the process running tests

Reply:
wait. let me confirm

Reply:
I just confirmed. when I remove dotnet from path the test still works. The reason being there is a dotnet file already in the execution directory